### ISSUE TITLE
solve error

### ISSUE THREAD

**Issue by Akwasi-Richie** [thieu1995/mealpy#54]
Hi @thieu1995 

I'm attempting to optimise a UNET model with GWO but I keep running into an error and the process terminates. the hyperparameters i am optimizing are the batch size, epoch. activation function, loss function, and learning rate.
I've defined and set my upper and lower bounds accordingly and set the mode to process since the network is a large one. however, after the first epoch which was 20, the process ends abruptly and throws an error.
I have attached snips of my code and error message to this issue.
kindly help me out

![c1](https://user-images.githubusercontent.com/70091983/160506418-782040c7-9265-4432-8358-b04d7ec1a349.png)
![c2](https://user-images.githubusercontent.com/70091983/160506422-4769b0b4-9dd9-4988-a4f6-eb278b6172b7.png)
![c3](https://user-images.githubusercontent.com/70091983/160506425-e6177050-9b3f-44c6-8fdd-291d3d807f36.png)

 and the error message 
![error](https://user-images.githubusercontent.com/70091983/160506526-6599b487-b638-4133-914d-0264718cd842.png)

**Comment by thieu1995** [comment]
Hi @Akwasi-Richie ,

Have you tried the latest version yet?

**Comment by Akwasi-Richie** [comment]
Yeah
The objective function kept running into an error although the older versions had no issue with the objective function. I gave up and used the default mode using the old versions. 

Took a long time but i got the results

**Comment by thieu1995** [comment]
@Akwasi-Richie ,

I think the problem come from the multi-processing module that you used in your model (resnet model or keras model). In mealpy, I also did use multi-processing module. I don't know but it will show error when both models are using multi-processing. 
You can try different training mode such as "thread" and "sequential" in the current version (2.4.0). If you problem is too big better to use "thread" training mode instead of "sequential".
